package.json

```diff
@@ -2,30 +2,12 @@
   "name": "sdlc-react",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.2.1",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2",
-    "keycloak-js": "^15.0.2",
-    "serve": "^13.0.2"
-  },
   "scripts": {
-    "start": "serve -s build -p 8080",
-    "devStart": "react-scripts start",
+    "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --coverage",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "eject": "react-scripts eject",
+    "lint": "eslint src --ext .ts,.tsx"
```

Comment:
change back `start` script to `serve -s build -p 8080` it's needed for build & deploying app with Paketo buildpacks

Reply:
done